### PR TITLE
add cropperjs v0.7.0 with npm autoupdate config

### LINKED ISSUE #7674: [Request] Add cropperjs
**Library name:** cropperjs
**Git repository url:** https://github.com/fengyuanchen/cropperjs

### PR DESCRIPTION
**Pull request by dannyxx001**
PR for #7674 
@pvnr0082t please help me review this PR, thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/fengyuanchen/cropperjs
 * Official website (optional, not the repository): http://fengyuanchen.github.io/cropperjs/
 * NPM package url (optional): https://www.npmjs.com/package/cropperjs
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 724
   - Count of watchers: 31
   - Count of forks: 75
 * NPM download stats (optional):
   - Downloads in the last day: 236
   - Downloads in the last week: 1161
   - Downloads in the last month: 4973

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

close #7674, cc @fengyuanchen

### CODE REVIEW

File: ajax/libs/cropperjs/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  "name": "cropperjs",
+  "filename": "cropper.min.js",
+  "description": "JavaScript image cropper.",
+  "version": "0.7.0",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fengyuanchen/cropperjs.git"
+  },
+  "homepage": "http://fengyuanchen.github.io/cropperjs",
```
Comment (pvnr0082t) [inline ajax/libs/cropperjs/package.json]: @dannyxx001 Please try to use https for github.io domain, most of them prefer https.

**Comment by dannyxx001** [comment]
OK, I had updated it. @pvnr0082t please review again, thanks.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
Thanks.

**Pull request merged**